easybuild/tools/package/package_naming_scheme/easybuild_pns.py

```diff
@@ -50,4 +49,4 @@ def version(self, ec):
             # cfr. http://rpm.org/ticket/56,
             # https://debian-handbook.info/browse/stable/sect.manipulating-packages-with-dpkg.html (see box in 5.4.3)
             ebver.replace('dev', '~dev')
-        return 'eb-%s' % ebver
+        return '%s~eb' % ebver
```

Comment:
What you should do is add a timestamp after the ~eb, as in ~eb-yyyymmdd to make it possible to update the eb built package automatically. Using the ```date_stamp = datetime.today().strftime('%Y%m%d')``` code from the first commit.